compiler/rustc_codegen_ssa/src/mir/block.rs

```diff
@@ -35,6 +35,14 @@ enum MergingSucc {
     True,
 }
 
+/// Indicates to the call terminator codegen whether a cal
```

Comment:
The final word on the line is misspelled - should be `call` if I'm not mistaken.

Reply:
I've included a fix of this typo in #144865